liferay-platform-experience/liferay-portal
# LPD-33175

Ticket: https://liferay.atlassian.net/browse/LPD-33175
## Motivation
There is no “X” button on Product menu when it shows Control Panel selection.

## Proposed Solution
Removed d-md-none from the "X" button so it is no longer hidden. 

## Steps to Verify
1. Start a clean bundle
2. Go to Control Panel > Instance Setting > Navigation
3. Uncheck “Enable Applications Menu” and Update
4. Go to HOME and open Product menu
5. Observe "X" button is missing

## Screenshots/videos (if applies)
Before:
![image](https://github.com/user-attachments/assets/90228d45-ce78-44bd-8ef2-df3fbb904209)

With fix applied: 
![image](https://github.com/user-attachments/assets/68db363c-2d1b-40a6-819d-babe2d5d7ed3)
